Code/DataStructs/BitOps.cpp

```diff
@@ -785,6 +785,7 @@ template void UpdateBitVectFromBinaryText(ExplicitBitVect&, const std::string&);
 #ifdef _MSC_VER
 #include <intrin.h>
 #define __builtin_popcount __popcnt
+#define __builtin_popcountl __popcnt64
```

Comment:
Re: compilers
I've built this with clang on the Mac, with MSVC on windows, and with g++ on linux. We do still need to test older versions of g++ (for RHEL5 compatibility), but worst case the popcnt stuff can always be turned off there.